Controller/SecurityController.php

```diff
@@ -26,6 +27,10 @@ class SecurityController extends Controller
      */
     public function loginAction(Request $request)
     {
+        if ($this->get('security.authorization_checker')->isGranted('IS_AUTHENTICATED_REMEMBERED')) {
```

Comment:
this is a no-go, because it breaks when Symfony asks a remembered user to login again to achieve full authentication